code/_onclick/hud/fullscreen.dm

```diff
@@ -23,19 +23,19 @@
 	return screen
 
 /mob/proc/clear_fullscreen(category, animated = 10)
+	set waitfor = 0
 	var/obj/screen/fullscreen/screen = screens[category]
 	if(!screen)
 		return
 
 	screens -= category
 
 	if(animated)
-		spawn(0)
-			animate(screen, alpha = 0, time = animated)
-			sleep(animated)
-			if(client)
-				client.screen -= screen
-			qdel(screen)
+		animate(screen, alpha = 0, time = animated)
+		sleep(animated)

```

Comment:
Should use a clienttime timer since animate() runs on client time and not byond time.